package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "warriors-frontend",
+  "version": "1.0.0",
+  "description": "Web application frontend built with React, for Andela Kigali Stack-Up 3",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server",
+    "start:dev": "npm run build:dev",
+    "build": "webpack --mode development",
+    "build:dev": "webpack -d && webpack-dev-server --content-base src/ --inline --hot",
+    "dev": "webpack --mode development --config webpack.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/STACK-UP-3/warriors-frontend.git"
+  },
+  "keywords": [
+    "react",
+    "barefoot nomad"
+  ],
+  "author": "Warriorz Team - Andela Kigali Stack-Up 3",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/STACK-UP-3/warriors-frontend/issues"
+  },
+  "homepage": "https://github.com/STACK-UP-3/warriors-frontend",
+  "dependencies": {
+    "materialize-css": "^1.0.0-rc.2",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-router-dom": "^5.2.0"
+  },
+  "devDependencies": {
```

Comment:
`devDependencies`: are dependencies that are only used in development thus will not be necessary for production, this means that they will not be installed in production.
And I think only _babel-eslint, webpack-cli, and webpack-dev-server_, are the ones which will not be required in production. 
Thus I think this could result in a problem when deploying the app in production.

Reply:
Ya I get you... Heroku does not install `devDependencies` 🤦🏽